app/db/index.js

```diff
@@ -43,29 +53,32 @@ function getSpecificMarket(market) {
  * @param {*} info All question answers from the assessment page.
  */
 function addNewMarket(info) {
-  let marketName = `${info.marketInfo.marketName}, ${info.marketInfo.address}`;
-  // Make sure illegal characters removed from key.
-  marketName = marketName.replace(/[^0-9a-zA-Z, ]/gi, '').trim();
+    let marketName = info.marketInfo.marketName + ', ' + info.marketInfo.address;
```

Comment:
extract this into its own function (`createMarketKey` or something) so it's a little clearer what's going on, also has the added benefit of being able to make `marketName` a constant